subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -103,6 +103,12 @@ enum {
 			       1),
 #endif /* CONFIG_BT_CONN */
 
+#if defined(CONFIG_BT_CTLR_CONN_ISO_GROUPS)
+	TICKER_ID_CIG_BASE,
```

Comment:
I would prefer the name `TICKER_ID_CONN_ISO_BASE`, but I can live with the current name if the other reviewers prefer it.